internal/graphql/resolver/operator.go

```diff
@@ -279,3 +300,23 @@ func applyYaml(client *mesherykube.Client, delete bool, file string) error {
 
 	return nil
 }
+
+func (r *Resolver) cleanEntries(del bool) error {
+	if del {
+		r.Log.Info("Removing Operator")
+		objs, err := getOperator(r.DBHandler)
+		if err != nil {
```

Comment:
@kumarabd is the Operator orphaned at this point? What should users do about this? Can you provide an probable cause and suggested remediation?